contents/newsletter/talk-to-users.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 8 things engineers should know about talking to users
+date: 2024-04-22
+author:
+  - ian-vanagas
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/newsletter/beyond-10x-engineer/super-hog.png
+featuredImageType: full
+tags:
+  - Engineering
+  - Product engineers
+crosspost:
+  - Product engineers
+  - Blog
+---
+
+Talk to users? Isn't that a job for founders, product managers, and salespeople? Why would an engineer ever want or need to do it?
+
+In his book *[Modern Software Engineering](https://www.davefarley.net/?p=352)*, Dave Farley writes that mastering software engineering requires becoming an expert in learning. When you prioritize learning, talking to users becomes a critical part of an engineer's toolkit.
+
+To explain further why this is the case, and to help you get started, we wrote this guide based on what we've learned talking to users as engineers ourselves at PostHog.
+
+## 1. Talking to users isn't a waste of time
+
+Many engineers place shipping code above all else. Anything that gets in the way of this is, apparently, a waste of time. This reflects in common metrics for measuring engineering success like lead time or deployment frequency.
+
+This creates a disregard for shipping the right code: features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that only one-third of ideas tested improved metrics they were designed for.
+
+Spending more time learning what users want is a fix for this. This helps close the gap between what users value and what you are building.
+
+![Don't build the wrong features](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470851/posthog.com/contents/images/newsletter/talk-to-users/features.jpg)
+
+## 2. You have an information bottleneck
+
+You might already think you know what users want. Someone spoke to users, turned those insights into tickets, and you to build them. The problem with this is there has been a lot of information lost in the process of going from interviews to tickets.
+
+![Information bottleneck](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/10x-engineers-do-user-interviews/pm-to-engineer.png)
+
+As an engineer, you miss out on the context and depth of information users can provide. The person talking to the user likely doesn't know the options and constraints as well as you do. You miss out on being able to dive deeper on those areas and use that information to close in on a good solution
+
+Think of user interviews like automated tests. Not doing them saves you time, but also leads to pain over the long term. Both enable you to iterate faster by providing rapid feedback on if what you've built is working, even if they aren't code you ship.
+
+## 3. Preparing to talk to users
```

Comment:
this applies to all the crossheads, but if the title is '8 things engineers should know about talking to users' then the crossheads need to be those things. so more like

3. You've got to prepare
4. Find the right users to talk to

just little tweaks to how the crossheads are phrased